contracts/merkle-distributor/AcrossMerkleDistributor.sol

```diff
@@ -61,4 +73,27 @@ contract AcrossMerkleDistributor is MerkleDistributor {
         require(whitelistedClaimers[msg.sender] || _claim.account == msg.sender, "invalid claimer");
         super.claim(_claim);
     }
+
```

Comment:
Maybe this should be done in a separate PR, but it's a little confusing that whitelisted claimers can claim directly to their own address _and_ call claimMulti to send tokens to the user.

If we end up going in this direction, do you think we should drop the overridden claimMulti and claim methods, just to reduce surface area and complexity?

Reply:
I think we still need to require that the claim.account is the caller so users can't reset each other's referral multipliers but i can remove the whitelist requirement. Thats more so to make testing easier but there's an easier way to do this

Reply:
Implemented the change where the whitelist only applies for `claimFor`: https://github.com/across-protocol/contracts-v2/pull/200/commits/e21526aab9f40fc1f1b6f03259905b627273dd29